modules/check_in/spec/serializers/check_in/v2/appointment_identifiers_serializer_spec.rb

```diff
@@ -32,7 +32,7 @@
             country: 'USA'
           },
           homePhone: '5552223333',
-          mobilePhone: '5553334444',
+          patientCellPhone: '5553334444',
```

Comment:
The current CHIP update has this field outside of the demographics object, since it is not part of the fetched demographics.

Reply:
It's in the options object see here: https://github.com/department-of-veterans-affairs/chip/pull/1895/files#diff-74d0e527efa21b5f9d890e31c1d7379c7e6dd79125b488e8a56b2b27b088748a